cleverhans/experimental/certification/dual_formulation.py

```diff
@@ -90,8 +90,8 @@ def __init__(self, sess, dual_var, neural_net_param_object, test_input, true_cla
       lo_plus_up = self.nn_params.forward_pass(self.lower[i] + self.upper[i], i)
       lo_minus_up = self.nn_params.forward_pass(self.lower[i] - self.upper[i], i, is_abs=True)
       up_minus_lo = self.nn_params.forward_pass(self.upper[i] - self.lower[i], i, is_abs=True)
-      current_lower = 0.5 * (lo_plus_up + lo_minus_up + self.nn_params.biases[i])
-      current_upper = 0.5 * (lo_plus_up + up_minus_lo + self.nn_params.biases[i])
+      current_lower = 0.5 * (lo_plus_up + lo_minus_up) + self.nn_params.biases[i]
+      current_upper = 0.5 * (lo_plus_up + up_minus_lo) + self.nn_params.biases[i]
```

Comment:
What is this change about?

Is it the bug in bounds computation which you mentioned before? If yes - it's not really related to this PR and it's better to put it into separate own PR.

Reply:
yes this was the bug. ok